routes/quiz.js

```diff
@@ -9,6 +9,7 @@ const { authMiddleware, restrictTo } = require('~/middlewares/auth')
 
 router.use(authMiddleware)
 
+router.get('/:id', asyncWrapper(quizController.getQuizById))
```

Comment:
use isEntityValid middleware

Reply:
or add this check inside of the quiz service

Reply:
> use isEntityValid middleware

fixed